test/parser.test.coffee

```diff
@@ -178,13 +178,13 @@ module.exports =
     equ r.sample.$$[10].$$[5]._, '6'
 
     # test text ordering with XML nodes in the middle
-    equ r.sample.$$[16]['#name'], 'textordertest'
```

Comment:
I added a new element, attrValueProcessTest, in test/fixtures/sample.xml (line 54).  In theory I could have re-purposed the existing attrNameProcessTest fixture, in which case the index for textordertest would not have changed, but it seems more appropriate to have separate fixtures for the separate tests.